streaming/index.js

```diff
@@ -996,7 +996,7 @@ const startServer = async () => {
   // @ts-expect-error
   api.use(errorMiddleware);
 
-  api.get('/api/v1/streaming/*', (req, res) => {
+  api.get(/\/api\/v1\/streaming/, (req, res) => {
```

Comment:
From what I understand from https://expressjs.com/en/guide/migrating-5.html#path-syntax we just need to name the splat pattern, e.g.:
```suggestion
  api.get('/api/v1/streaming/*splat', (req, res) => {
```